### PR TITLE
Fixes #32227 - Do not enhance Jenkins rake task

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: lib/tasks/foreman_remote_execution_tasks.rake [hunk 1]
```diff
@@ -38,8 +38,3 @@ namespace :foreman_remote_execution do
 end
 
 Rake::Task[:test].enhance ['test:foreman_remote_execution']
-
-load 'tasks/jenkins.rake'
-if Rake::Task.task_defined?(:'jenkins:unit')
-  Rake::Task['jenkins:unit'].enhance ['test:foreman_remote_execution', 'foreman_remote_execution:rubocop']
```
Comment (adamruzicka) [inline lib/tasks/foreman_remote_execution_tasks.rake:44]: My understanding of the linked issue was that only rubocop is problematic, not tests. Why remove both?

**Comment by xprazak2** [comment]
I kept the tests, removing only rubocop now.

**Review by adamruzicka — APPROVED** [review]
I'm fine with this, ACK

**Comment by adamruzicka** [comment]
Thank you @xprazak2 !

**Comment by xprazak2** [comment]
Thanks, any plans on a new rex release so we can get the tests green on openscap?

**Comment by adamruzicka** [comment]
So far there are almost no changes since the last release, I'd like to get another PR or two merged before cutting a new one.

**Comment by adamruzicka** [comment]
@xprazak2 version 4.3.1 containing this fix was just released

**Pull request merged**